app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
```

Comment:
The `people` attribute is defined as a class attribute, which means it is shared across all instances of the `Person` class. This can lead to unexpected behavior when creating multiple `Person` instances. Consider using an instance attribute instead to ensure each `Person` has its own dictionary.